### PR TITLE
Rewrite extensions from xml to xsd

### PR DESCRIPTION
**Pull request by jasollien**

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<projectDescription>
+	<name>BCF-XML</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+	</natures>
+</projectDescription>
```
Comment (ykulbak) [inline .project:11]: @jasollien  Can you please exclude the project file?

**Comment by ykulbak** [comment]
@jasollien looks ok to me, can you please resubmit without the changes that duplicate #235 ?

File: .project (hunk 1 above)
Comment (jasollien) [reply .project:11]: sorry. Auto generated from eclipse. Will remove

**Comment by jasollien** [comment]
@ykulbak: done

**Review by pasi-paasiala — APPROVED** [review]
+1

**Pull request merged**